### ISSUE TITLE
`gem install metanorma-cli` fails

### ISSUE THREAD

**Issue by strogonoff** [metanorma/metanorma-cli#119]
On fresh install of Ubuntu 18.04 I’m getting the following:

```console
ubuntu@ip-172-31-5-74:~$ sudo gem install metanorma-cli
ERROR:  Could not find a valid gem 'metanorma-iec' (~> 0.0.5) in any repository
ERROR:  Possible alternatives: metanorma-iec
ubuntu@ip-172-31-5-74:~$ 
```

Since it’s one of the official ways to install Metanorma in project’s docs, filing as “bug”

**Comment by ronaldtse** [comment]
That's so strange. @opoudjis could you help here? I think `metanorma-iec` 0.0.5 is NOT released.

```
$ ~/src/latexml-snap (master): gem install metanorma-iec
Successfully installed metanorma-iec-0.0.4
1 gem installed
```

**Comment by ronaldtse** [comment]
@strogonoff : @opoudjis has just released `metanorma-iec` version `0.0.5`. Closing this for now.
